packages/algolia/src/client.ts

```diff
@@ -88,6 +91,7 @@ export class AlgoliaSearchClient {
     entityTypes: T[],
     query: string,
     requestOptions?: SearchOptions,
+    descending?: boolean,
```

Comment:
what if we need another replica for another different sort type? Are these replicas there just so we can search in reverse order? I think other combinations might be required later, maybe we need some sort of map here for sort order? also technically at the moment this replica should only ever be used for events - shouldnt our type or implementation reflect that?